modules/ROOT/pages/source/payload-mode.adoc

```diff
@@ -0,0 +1,185 @@
+    "LSP": null
+  },
+  "timestamp": {
+    "type": "I64",
+    "B": null,
+    "I64": 1729779365447,
+    "F64": null,
+    "S": null,
+    "BA": null,
+    "TLD": null,
+    "TLDT": null,
+    "TLT": null,
+    "TZDT": null,
+    "TOT": null,
+    "TD": null,
+    "SP": null,
+    "LB": null,
+    "LI64": null,
+    "LF64": null,
+    "LS": null,
+    "LTLD": null,
+    "LTLDT": null,
+    "LTLT": null,
+    "LZDT": null,
+    "LTOT": null,
+    "LTD": null,
+    "LSP": null
+  }
+}
+
+----
+
+This mode is especially beneficial for data with complex schema requirements, as it ensures compatibility even if property types change on the Neo4j side.
+
+== Understanding the `EXTENDED` Payload Structure
+
+In `EXTENDED` mode, each property includes fields for every supported Neo4j type. Only the field corresponding to the actual property type will contain a non-null value, while all others are set to null. This structure ensures that any change in the type of a property does not cause schema enforcement errors at either the source or sink connector.
+
+[cols="1,2"]
+|===
+| Field  | Description
+
+| type   | Indicates the type of the property. Possible values include: `B`, `I64`, `F64`, `S`, `BA`, `TLD`, `TLDT`, `TLT`, `TZDT`, `TOT`, `TD`, `SP`, or their list equivalents (e.g., `LB`, `LI64`, `LF64`, `LS`, `LTLD`, etc.).
+| B      | Boolean type (true or false)
+| I64    | 64-bit integer
+| F64    | 64-bit floating point
+| S      | String
+| BA     | Byte array
+| TLD    | Temporal Local Date
+| TLDT   | Temporal Local DateTime
+| TLT    | Temporal Local Time
+| TZDT   | Temporal Zoned DateTime
+| TOT    | Temporal Offset Time
+| TD     | Temporal Duration
+| SP     | Spatial Point
+| LB, LI64, LF64, LS, LTLD, etc. | Lists of each corresponding type
+|===
+
+For example, a string field will be represented as:
+
+[source,json]
+----
+{
+  "type": "S",
+  "B": null,
+  "I64": null,
+  "F64": null,
+  "S": "actual_value",
+  ...
+}
+----
+
+== Configuration Recommendations
+
+`COMPACT` mode is useful and easier to work with when generated messages are consumed with other connectors or applications, and you can relax your schema compatibility mode on target topics. If your environment requires schema compatibility, temporal data types, or you have strong type safety requirements with different converters (`AVRO`, `JSON Schema`, `PROTOBUF` or `JSON Embedded`), `EXTENDED` mode should be preferred.
+
+== Compatibility with Sink Connectors
+
+The `EXTENDED` format was introduced in connector version 5.1.0 to ensure that all data published to Kafka topics adheres to a consistent schema. This prevents issues when a property changes type on the Neo4j side (e.g., a name property changes from integer to string), enabling smooth data processing across connectors and Kafka consumers. When a Neo4j sink connector is fed by a Neo4j source connector, it’s recommended to use `EXTENDED` mode, as the Neo4j sink connector can seamlessly handle the `EXTENDED` data type.
```

Comment:
do `COMPACT` source data work with sinks in some circumstances?
if it never works, I'd use stronger language in the last sentence. Something like "`EXTENDED` mode must be used with sinks..."